src/script/scripting_server.cpp

```diff
@@ -89,28 +89,29 @@ ServerScripting::ServerScripting(Server* server):
 	infostream << "SCRIPTAPI: Initialized game modules" << std::endl;
 }
 
-void ServerScripting::initAsync()
+void ServerScripting::saveGlobals()
 {
-	// Save globals to transfer
-	{
-		lua_State *L = getStack();
-		lua_getglobal(L, "core");
-		luaL_checktype(L, -1, LUA_TTABLE);
-		lua_getfield(L, -1, "get_globals_to_transfer");
-		lua_call(L, 0, 1);
-		auto *data = script_pack(L, -1);
-		assert(!data->contains_userdata);
-		getServer()->m_async_globals_data.reset(data);
-		lua_pushnil(L);
-		lua_setfield(L, -3, "get_globals_to_transfer"); // unset function too
-		lua_pop(L, 2); // pop 'core', return value
```

Comment:
This was not moved to the new code. Which function does pop `core` now?

Reply:
`top` points to the core table and any handling of that is done by the caller somewhere else

Reply:
An explicit pop would be cleaner imo.